### PR TITLE
Add basic instrumentation for ReadRows calls

### PR DESCRIPTION
**Pull request by emkornfield**
For now I've only instrumented the Arrow DSv2 path.  As needed we can potentially add more paths and collect different metrics.

This is a strawman proposoal, happy to make changes if we think there is a better approach.

CC @davidrabinowitz

### CODE REVIEW

File: connector/src/main/java/com/google/cloud/spark/bigquery/v2/ArrowInputPartition.java [hunk 1]
```diff
@@ -15,9 +15,7 @@
  */
 package com.google.cloud.spark.bigquery.v2;
 
-import com.google.cloud.bigquery.connector.common.BigQueryReadClientFactory;
-import com.google.cloud.bigquery.connector.common.ReadRowsHelper;
-import com.google.cloud.bigquery.connector.common.ReadSessionResponse;
+import com.google.cloud.bigquery.connector.common.*;
```
Comment (davidrabinowitz) [inline connector/src/main/java/com/google/cloud/spark/bigquery/v2/ArrowInputPartition.java:18]: Please avoid `.*` imports

**Review by davidrabinowitz — APPROVED** [review]

File: connector/src/main/java/com/google/cloud/spark/bigquery/v2/ArrowInputPartition.java (hunk 1 above)
Comment (emkornfield) [reply connector/src/main/java/com/google/cloud/spark/bigquery/v2/ArrowInputPartition.java]: sorry about that, this is a fresh intellij install.  Is there a style file I should import?

Comment (davidrabinowitz) [reply connector/src/main/java/com/google/cloud/spark/bigquery/v2/ArrowInputPartition.java]: Not yet unfortunately... Once we will have the new architecture and the move to maven I will integrate a [google-java-format](https://github.com/google/google-java-format#third-party-integrations) maven plugin.

**Pull request merged**